test/BuiltIns/rlexe.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <files>ChakraLib.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>BuiltIn LdChakraLib</tags>
```

Comment:
Technically LdChakraLib should always be turned off except when serializing the bytecode for the builtins if I understood correctly